<launch>
  <node type="rviz" name="rviz" pkg="rviz" args="-d /home/rodrigo/catkin_ws/src/TFM_Landmark_Based_Localization/clustering_debug.rviz" />
  <node pkg="pointcloud_clustering" type="waymo_client.py" name="waymo_client" output="screen"/>
  <node pkg="pointcloud_clustering" type="clustering" name="clustering" output="screen"/>
    <param name="config_file_path" value="$(find pointcloud_clustering)/config/config.conf"/>
  <node pkg="pointcloud_clustering" type="landmark_detection.py" name="landmark_detection" output="screen"/>
  <!-- <node pkg="pointcloud_clustering" type="data_fusion" name="data_fusion" output="screen"/>
    <param name="data_fusion_config_file_path" value="$(find pointcloud_clustering)/config/data_fusion_config.conf"/>
    <param name="data_fusion_map_file_path" value="$(find pointcloud_clustering)/map/map.json"/> -->
</launch>